packages/web3-eth-ens/src/ens.ts

```diff
@@ -39,6 +39,9 @@ import { registryAddresses } from './config';
 import { Resolver } from './resolver';
 
 export class ENS extends Web3Context<EthExecutionAPI & Web3NetAPI> {
```

Comment:
We need a description for the class, please.

Reply:
Additionally, comment for the constructor for its parameters and sample usage, please.